src/components/SmallInfoCard.tsx

```diff
@@ -1,38 +1,58 @@
 import { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import Title from './Title';
 
 import api from 'src/service/api';
 
 import buttonImgSrc from '../assets/image95.png';
 import '../styles/SmallCardInfo.scss';
 
-const SmallInfoCard = () => {
+type CardinfoProps = {
+  id: number;
+};
 
-  const [data, setData] = useState<{}>();
+type DataType = {
+  title: string;
+  poster_path: string;
+  vote_average: string;
+  vote_count: string;
+  id?: number;
+};
```

Comment:
better to avoid abstract names - please rename to reflect real purpose of this type, no need to add Type to name